cli.js

```diff
@@ -274,13 +274,19 @@ program
   .option('-l, --limit <n>', 'Only take limit instances from the file', x =>
     parseInt(x, 10)
   )
+  .option(
+    '-p, --pool-size <n>',
+    'Limit the number of instances on which we execute the script at the same time',
```

Comment:
do we keep the readme in sync with all our commands? Or do we redirect to ach -h?

Reply:
there is no README entry for `ACH batch` so for now I think that `ACH batch -h` is good enough

Reply:
ok